src/api/api.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/F1QNPxJu7Anv36JZ66mG/books/';
+
+const getBooks = async () => {
+  const response = await axios.get(url);
+  const { data } = response;
+  const books = [];
+  Object.entries(data).forEach((book) => {
+    books.push(
+      {
+        title: book[1][0].title,
+        category: book[1][0].category,
+        item_id: `${book[0]}`,
+      },
+    );
+  });
+  return books;
+};
+
+const createBook = async (newBook) => {
+  const response = await axios.post(url, newBook);
+  return response.data;
+};
+
+const removeBook = async (id) => {
+  const removeURL = `${url}${id}`;
+  const response = await axios.delete(removeURL);
+  return response.data;
+};
+
+export { getBooks, createBook, removeBook };
```

Comment:
- [ ] Connect to API is not working properly, check the following points:

     - You need to render the fake data you have setted once you start a server.
     - The POST or Added data should be stored in the redux store correctly,.
     - The redux store should keep the information but for you once I reload the page the data are being deleted or removed from the screen so make sure that the user lost his or her data once he or she removes the book.
     
   - [ ] Data from the API requests should be stored correctly in Redux.
   
   - [ ] The redux code of the application should be refactored using `middleware` to make async requests to the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739).
   
   - [ ] Server requests should made in the componentDidMount lifecycle method or with react hooks.
     
   - [ ] The Add book and remove book features should work the same as before even after the refactoring to use the API has been done. Add and Remove book button works perfectly however, make sure to keep the added book in the redux store correctly.